acceptance/bundle/templates/default-python/integration_classic/output.txt

```diff
@@ -54,46 +54,39 @@ Resources:
 >>> diff.py ../out.validate.dev.json ../out.summary.dev.json
 --- ../out.validate.dev.json
 +++ ../out.summary.dev.json
-@@ -5,4 +5,6 @@
-       "files": [
-         {
-+          "patched": "",
-+          "remote_path": "",
-           "source": "[TEST_TMP_DIR]/project_name_[UNIQUE_NAME]/dist/*.whl"
-         }
-@@ -50,4 +52,5 @@
+@@ -50,4 +50,5 @@
```

Comment:
Interesting that out.validate.dev.json already did not include those patched and remote_path fields. Looking closer I see that "bundle summary" renders b.Config while "bundle validate" renders b.Config.Value.AsAny().